### DISCUSSION TITLE
Old Dates

### DISCUSSION THREAD

**Discussion by Sarakilani** [sql-bi/Bravo#448] [Q&A] [answered]
Hello,

How can I make the calendar begin from the minimum date I have? I also faced this same issue before using Bravo, when I used CALENDARAUTO() in DAX.

Thanks

**Comment by albertospelta** [comment, marked as answer]
@Sarakilani You can configure the date range within the 'Interval' panel. If you leave the `First Year` parameter empty, an automatic scan will be applied and the minimum date in your datamodel is used as first year.

![image](https://user-images.githubusercontent.com/1957788/174744980-9772b44b-542b-4bef-87c1-47e40c3aaca0.png)

**Reply by Sarakilani** [reply]
Thanks :)
